lms/static/scripts/frontend_apps/utils/grader-service.js

```diff
@@ -1,13 +1,31 @@
 import { apiCall } from './api';
 
 /**
- * Submits the grade to the LTI endpoint, waits for it to return, then updates the state with the
- * new grade
+ * @typedef {Object} Student
+ * @prop {string} LISResultSourcedId - The unique student id
```

Comment:
Technically, this isn't a student ID but a unique identifier to the outcome for this student/assignment combination. It can change over time (the LMS backend stores a fresh copy each time the student launches an evaluation-enabled assignment). So, calling it a "unique outcome identifier" might be more accurate.

Reply:
will do, thanks!